cfme/control/explorer/actions.py

```diff
@@ -112,7 +112,47 @@ def is_displayed(self):
         )
 
 
-class Action(Updateable, Navigatable, Pretty):
+class ActionCollection(BaseCollection):
+
+    def __init__(self, appliance):
+        self.appliance = appliance
+
+    def instantiate(self, description, action_type, action_values=None):
+        return Action(self, description, action_type, action_values=action_values)
+
+    def create(self, description, action_type, action_values=None):
+        """Create an Action in the UI."""
+        action = self.instantiate(description, action_type, action_values=action_values)
```

Comment:
If action_values keys and values correspond to the view, then you should be able to move this to the end of this method, which would be preferable.